script.js

```diff
@@ -243,3 +243,47 @@ ${dynamicImage}
     }
   });
 }
+
+// Javascript for the local storage
+const userDataInLocalStorage = localStorage.getItem('userData');
+const parsedData = JSON.parse(userDataInLocalStorage);
+document.getElementById('fullName').value = parsedData.name;
+document.getElementById('email').value = parsedData.email;
+document.getElementById('textArea').value = parsedData.comment;
+function localStorageMine() {
+  const userName = document.getElementById('fullName');
+  const userEmail = document.getElementById('email');
+  const userComment = document.getElementById('textArea');
+  const uname = userName.value;
+  const email = userEmail.value;
+  const comment = userComment.value;
+  const inputFieldObject = {
+    name: uname,
+    email,
+    comment,
+  };
```

Comment:
According to the [project requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/10_localstorage.md), please let's implement the following interactions and ensure that everything works fine. 💪

![image](https://github.com/ZabihullahNooriWardak/Zabih-portfolio/assets/78251927/90a605af-73db-4edb-88fd-6e4227828851)

Currently, your app is not working as expected, and the data from the `form` are not saved to the local storage. 👀

Error in the console
![image](https://github.com/ZabihullahNooriWardak/Zabih-portfolio/assets/78251927/1e133219-a4c8-4138-8fc1-d4f715c7d8c8)

Source
![image](https://github.com/ZabihullahNooriWardak/Zabih-portfolio/assets/78251927/53eac1f0-a768-42e3-b5da-e3cd3eb88820)